lib/cartopy/crs.py

```diff
@@ -3219,6 +3219,59 @@ def x_limits(self):
     def y_limits(self):
         return self._y_limits
 
+class Spilhaus(Projection):
+    """
+    Spihaus-Adams World Ocean Map in a Square II
+
+    """
+    def __init__(self,greenland_at = 0,false_easting = 0.0,false_northing = 0.0):
+        """
+        Parameters
+        ----------
+        greenland_at: optional
+            Orientation of the map. 0,1,2,3, stand for Greenland at upper left,
+            upper right, lower right and lower left, respecivly. Defaults to 0.
+            Strings like 'upper left' are also allowed.
+        false_easting: optional
```

Comment:
There should be a space before and after the ":" in these docstrings for sphinx to format it properly.
```suggestion
        false_easting : optional
```